### ISSUE TITLE
ios support

### ISSUE THREAD

**Issue by pablomaurer** [sdkcarlos/cordova-ourcodeworld-sftpplugin#1]
looks good =) thanks for sharing. Issue to track ios support.

**Comment by sdkcarlos** [comment]
Hi @mnewmedia thanks to you for checkup the plugin, pitifully i'm unable to work on objective-c because i don't use apple product (if someone has the possibility please check out sftp libraries for ios here : [sftp Libraries ios](http://stackoverflow.com/questions/14462257/sftp-libraries-for-iphone) ).

I have read about smething about "hackintosh" for windows to develop on  , but doesn't seems legal at all.

**Comment by pablomaurer** [comment]
Yep, the easiest way to use mac on windows, is to use vmware with image available in torrents. If you will buy a osx license, it will maybe be legal or at least a bit more legal. 

I have apple devices but no idea about Objective-C and have not the time at the moment to play with it. :'(
